### PR TITLE
Feature/family indicators

### PR DESCRIPTION
**Pull request by bhylak**
Added a view of family indicators that allows the social worker to switch between the snapshots. (And added the logic to view model to support such an endeavor)

### CODE REVIEW

File: app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/FamilyDetailFrag.java [hunk 1]
```diff
@@ -83,8 +84,9 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         Uri uri = Uri.parse("https://bongmendoza.files.wordpress.com/2012/08/urban-poor-family.jpg");
         mFamilyImage.setImageURI(uri);
 
-        Button surveyButton  = view.findViewById(R.id.survey_button);
+      //  Button surveyButton  = view.findViewById(R.id.survey_button);
```
Comment (krconv) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/FamilyDetailFrag.java]: Should these be left commented out?

File: app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/FamilyIndicatorsSubtab.java [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+                {
+                    mLevelIndicator.setBackgroundTintList(ContextCompat.getColorStateList(itemView.getContext(), color));
+                }
+            }
+        }
+    }
+
+    //The problem with creating a custom adapter (https://stackoverflow.com/a/8116756) is that the spinner
+    //has to be restyled.. it should be done eventually so we don't have to do the nonsense in this constructor
+    //TODO @blhylak clean up this mess
```
Comment (krconv) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/FamilyIndicatorsSubtab.java]: Todo left

File: app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/callbacks/SubTabFragmentCallback.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package org.fundacionparaguaya.advisorapp.fragments.callbacks;
+
+/**
+ * Created by benjaminhylak on 1/30/18.
```
Comment (krconv) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/callbacks/SubTabFragmentCallback.java]: Left default class comment

File: app/src/main/java/org/fundacionparaguaya/advisorapp/models/IndicatorQuestion.java [hunk 4]
```diff
@@ -49,4 +51,26 @@ public int hashCode() {
...
+                return this.getDescription().compareTo(other.getDescription());
+            }
+            else //sort by dimension
+            {
+                return this.getIndicator().getDimension().compareTo(other.getIndicator().getDimension());
+            }
+        }
+        catch (ClassCastException e)
+        {
+            Log.e("", e.getMessage());
```
Comment (krconv) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/models/IndicatorQuestion.java]: This should be a tag with class name

File: app/src/main/java/org/fundacionparaguaya/advisorapp/viewmodels/FamilyInformationViewModel.java [hunk 5]
```diff
@@ -1,33 +1,130 @@
...
 
-import java.util.concurrent.ExecutionException;
+import java.util.*;
+
+import static org.fundacionparaguaya.advisorapp.models.IndicatorOption.Level.Green;
+import static org.fundacionparaguaya.advisorapp.models.IndicatorOption.Level.Red;
+import static org.fundacionparaguaya.advisorapp.models.IndicatorOption.Level.Yellow;
 
 
 public class FamilyInformationViewModel extends ViewModel {
```
Comment (krconv) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/viewmodels/FamilyInformationViewModel.java]: This class has debug stuff left in it

File: app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/FamilyDetailFrag.java (hunk 1 above)
Comment (bhylak) [reply app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/FamilyDetailFrag.java]: [ ]

**Pull request merged**